redbot/cogs/trivia/trivia.py

```diff
@@ -512,6 +568,66 @@ def get_trivia_list(self, category: str) -> dict:
             else:
                 return dict_
 
+    async def _save_trivia_list(self, ctx: commands.Context, file: discord.Attachment):
+        """Checks and saves a trivia list to data folder.
+
+        Parameters
+        ----------
+        file : discord.Attachment
+            A discord message attachment.
+
+        Returns
+        -------
+        None
+        """
+        filename = file.filename.rsplit(".", 1)[0]
+
+        buffer = io.BytesIO(await file.read())
+        yaml.safe_load(buffer)
+        buffer.seek(0)
+
+        file = cog_data_path(self) / f"{filename}.yaml"
+
+        basefileexists = None
+
+        if filename in ("upload", "delete"):
+            basefilexists = True
+        elif any(filename == item.stem for item in get_core_lists()):
+            basefileexists = True
+
+        if file.exists() or basefileexists:
+            if basefileexists:
+                await ctx.send(
+                    _(
+                        "{filename} is a reserved trivia name and cannot be replaced.\n"
+                        "Choose another name".format(filename=filename)
+                    )
+                )
+                return
+            else:
+                await ctx.send(
+                    _(
+                        "{filename} already exists. Do you wish to overwrite?".format(
+                            filename=filename
+                        )
+                    )
+                )
+            pred = MessagePredicate.yes_or_no(ctx)
+            await ctx.bot.wait_for("message", check=pred)
+            if pred.result is True:
+                with file.open("wb") as fp:
+                    fp.write(buffer.read())
+                await ctx.send(
+                    _("Success, trivia list uploaded as {filename}.").format(filename=filename)
+                )
+            else:
+                await ctx.send(_("I am not replacing the existing file"))
+        else:
+            with file.open("wb") as fp:
+                fp.write(buffer.read())
+            LOG.debug("Saved file as {filepath}".format(filepath=file))
+            await ctx.send("Saved Trivia list as {filename}".format(filename=filename))
```

Comment:
This string has to be wrapped in `_()` for translations.
```suggestion
            await ctx.send(_("Saved Trivia list as {filename}").format(filename=filename))
```